node_modules/oae-util/lib/cassandra.js

```diff
@@ -347,14 +271,53 @@ var _createColumnFamilies = function(keys, families, callback) {
  * @param  {Function} callback      Standard callback function
  * @param  {Object}   callback.err  An error that occurred, if any
  */
-var runQuery = module.exports.runQuery = function(query, parameters, callback) {
+var runQuery = module.exports.runQuery = function(query, parameters, opts, callback) {
+    if (typeof opts === 'function') {
+        callback = opts;
+        opts = {
+            'cache': true
+        };
+    }
```

Comment:
I see what's being done here. I'm ok w/ it I think. Docs do need to be updated, and I wonder if there is a good way to highlight this style of function in the docs.

If we can, however, get rid of this parameter entirely with my previous comment, I would be in favour of that instead.